as2lib.py

```diff
@@ -131,7 +131,21 @@ def build_mdn(message, status, **kwargs):
     try:
         hparser = HeaderParser()
         message_header = hparser.parsestr(message.headers)
+
+        # default message
         text = _(u'The AS2 message has been processed. Thank you for exchanging AS2 messages with Pyas2.')
+
+        # check for default orginization message
+        if message.organization.confirmation_message:
+            text = message.organization.confirmation_message
+
+        # overwrite with partner specific message
+        if message.partner.confirmation_message:
+            text = message.partner.confirmation_message
+
+        if text == None:
```

Comment:
It was meant to check for null or blank values that could have gotten past Django. I have a more elegant solution to this now. I can create a new pull request before or after you merge.